src/themes/material/assets/mat.css

```diff
@@ -69,6 +69,20 @@ a:hover .article-title {
     word-break: break-all;
 }
 
+.article-body ul:not(.browser-default) {
+    display: block !important;
+    list-style-type: disc !important;
+    margin-block-start: 1em !important;
+    margin-block-end: 1em !important;
+    margin-inline-start: 0px !important;
+    margin-inline-end: 0px !important;
+    padding-inline-start: 40px !important;
```

Comment:
Do all these really need to be `!important`? we might want to override the ul styling on some journals/articles

Reply:
They do as we are overriding materialize.min.css that we do not have access to.

Reply:
Weird, I would have thought our specificity is higher than whichever rule is in materialize.min.css